linera-graphql-client/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "linera-graphql-client"
+version = "0.4.0"
+edition = "2021"
+description = "GraphQL client for the Linera network"
```

Comment:
If I understand correctly, this is _not_ a client one can use to talk to the validators—that's what I'd understand by "client for the Linera network". Rather, it's a library that facilitates talking to the `linera` client's GraphQL node service?

Reply:
yes